cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx

```diff
@@ -7,6 +7,7 @@ import { connect } from 'react-redux';
 
 import { CombinedState, ObjectType } from 'reducers/interfaces';
 import { createAnnotationsAsync, rememberObject } from 'actions/annotation-actions';
+// eslint-disable-next-line max-len
```

Comment:
In develop for cvat-ui we use ``'max-len': ['error', { code: 120, ignoreStrings: true }],``, so, I am not sure we need this command here